projects/igniteui-angular/src/lib/grids/hierarchical-grid/hierarchical-grid.integration.spec.ts

```diff
@@ -52,10 +60,10 @@ describe('IgxHierarchicalGrid Integration #hGrid', () => {
 
             expect(document.querySelectorAll('igx-grid-header').length).toEqual(3);
 
-            const firstRow = hierarchicalGrid.dataRowList.toArray()[0];
-            // first child of the row should expand indicator
-            (firstRow.nativeElement.children[0] as HTMLElement).click();
-            fixture.detectChanges();
+            const firstRow = hierarchicalGrid.dataRowList.first;
+            // the first row's cell should contain an expand indicator
+            expect(firstRow.nativeElement.children[0].classList.contains('igx-grid__hierarchical-expander')).toBeTruthy();
```

Comment:
You can create a constant for the 'igx-grid__hierarchical-expander', or to create a function in the HierarchicalGridFunctions to check if a row is expanded